express/server.js

```diff
@@ -37,22 +37,26 @@ const DB = require('./db');
 const express = require('express');
 const app = express();
 const cors = require('cors');
+var headers = {}
 const sentryEventContext = function(req, res, next) {
   const se = req.headers.se;
   if(!['undefined'].includes(se)) {
     Sentry.setTag("se", se);
+    headers["se"]=se
   }
 
   const customerType = req.headers.customertype;
   if(!['undefined'].includes(customerType)) {
     Sentry.setTag("customerType", customerType);
+    headers["customerType"]=customerType
   }
 
   const email = req.headers.email;
   if(!['undefined'].includes(email)) {
     Sentry.setUser({ 'email': email })
+    headers["user"]=email
```

Comment:
Could you make it `headers["email"]` instead? As this is what ruby [is looking for](https://github.com/sentry-demos/application-monitoring/blob/master/ruby/main.rb#L34). 'email' is always what the React app [sets it as](https://github.com/sentry-demos/application-monitoring/blob/master/react/src/components/Products.js#L35). If there is code on the frontend setting it as 'user' then please let me know and I'll have a look.

Reply:
@gid-sentry

Reply:
Done in [13c1ca8](https://github.com/sentry-demos/application-monitoring/pull/86/commits/13c1ca8dd10d5b017588278df804b170089ff807)